spec/tools/fix_auth/auth_config_model_spec.rb

```diff
@@ -24,11 +26,11 @@
         :type    => 'MiqProvisionRequest',
         :options => YAML.dump(
           :dialog                  => {
-            :'password::special' => enc_v1,
+            :'password::special' => enc_old,
           },
-          :root_password           => enc_v1,
-          :sysprep_password        => enc_v1,
-          :sysprep_domain_password => enc_v1
+          :root_password           => enc_old,
+          :sysprep_password        => enc_old,
+          :sysprep_domain_password => enc_old
```

Comment:
we may want to change one of these to a newly encoded value.

I remember having a case where some of the values were encrypted with the new value and others encrypted with the old value. Think this was a multi region change. can't remember the details.

I'm pretty sure this code can handle this and the code changes you made in miq-password also allow this use case

Reply:
Oh yeah, I can do that to ensure that works.